scripts/generators/beats.py

```diff
@@ -0,0 +1,86 @@
+import yaml
+from collections import OrderedDict
+
+
+def generate(ecs_nested, ecs_version):
+    # base first
+    beats_fields = fieldset_field_array(ecs_nested['base']['fields'])
+
+    allowed_fieldset_keys = ['name', 'title', 'group', 'description', 'footnote', 'type']
```

Comment:
Is the `group` key used for anything in Beats? If not then we could drop this field.

Reply:
Yes it comes from Beats.` group:1` fields are sorted before `group:2` fields, as far as I know.

Base is the only fieldset that has it set= to something other than 2 in ECS, and the semantics are different (base fields are not nested). So we could get rid of it eventually.

But group is present in Beats in some places, when it's not there, the default is 2.